crates/matrix-sdk-base/src/rooms/normal.rs

```diff
@@ -1430,6 +1463,44 @@ mod tests {
         assert!(info.base_info.topic.is_none());
     }
 
+    #[async_test]
+    async fn when_set_notable_tags_is_called_then_notable_tags_subscriber_is_updated() {
+        let (_, room) = make_room(RoomState::Joined);
+        let (_, mut notable_tags_subscriber) = room.notable_tags_stream().await;
+        stream_assert::assert_pending!(notable_tags_subscriber);
+        let notable_tags = RoomNotableTags::new(None);
+        room.set_notable_tags(notable_tags);
+        use futures_util::FutureExt as _;
+        assert!(notable_tags_subscriber.next().now_or_never().is_some());
+        stream_assert::assert_pending!(notable_tags_subscriber);
```

Comment:
here come the blank lines... :grin: 

```suggestion
        let (_, mut notable_tags_subscriber) = room.notable_tags_stream().await;
        
        stream_assert::assert_pending!(notable_tags_subscriber);
        
        let notable_tags = RoomNotableTags::new(None);
        room.set_notable_tags(notable_tags);
        
        use futures_util::FutureExt as _;
        assert!(notable_tags_subscriber.next().now_or_never().is_some());
        
        stream_assert::assert_pending!(notable_tags_subscriber);
```